Update FAQ answer

sherenem opened the pull request:
Resolves https://github.com/h2oai/h2o-llmstudio/issues/340

pascal-pfeiffer reviewed (approved):
lgtm

I believe there was a reason to remove the hardware question/answer with this PR?

sherenem commented:
@pascal-pfeiffer Sorry, that was unintentional. I have added it back. 

Also added in the link to the LLM Data Studio blogpost and removed a broken tooltip reference to a tooltip that had been removed with [this PR](https://github.com/h2oai/h2o-llmstudio/commit/be5ea884737e57e004367a0362c6e1d56df5ec3d#diff-3b50b9fde6b604b4e1b66eadac7a6522940300b4474a1fb5004fcfda62fc2a30).
